src/jestsetup.tsx

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2018 Ultimaker B.V.
+/* eslint-disable */
```

Comment:
`jestsetup.tsx` exists in multiple applications. Just noticed that for some applications `/* eslint-disable */` is not needed, as the dependencies are installed not only for dev, but also for production. Just want to check if there's any (in)consistency in these projects...

Reply:
I've looked into this… https://github.com/Ultimaker/stardust-web-components/pull/31/files#diff-e4403a877d80de653400d88d85e4801aR5 seems to be a nicer way of solving this without having to use inline overrides.